packages/tanstack-start/src/server/utils/index.ts

```diff
@@ -53,3 +53,27 @@ export function getResponseClerkState(requestState: RequestState, additionalStat
     headers: requestState.headers,
   };
 }
+
+/**
+ * Patches request to avoid duplex issues with unidici
+ * For more information, see:
+ * https://github.com/nodejs/node/issues/46221
+ * https://github.com/whatwg/fetch/pull/1457
+ * @internal
+ */
+export const patchRequest = (request: Request) => {
```

Comment:
@octoper Would it make sense or is it possible to have this be part of createClerkRequest ? 

I remember having to do something similar for remix.

Reply:
Yeah need need to revise in the `@clerk/backend` how we handle request, I will a ticket so we can have a better look at it

Reply:
Curious if adding this to `createClerkRequest` would break other SDKs like Next

Reply:
Yeah, I was not sure about it, I created a ticket so we can have a better look at it and have one solution for everything